### PR TITLE
Translated Version of the boilerplate's interface to English

### PR DESCRIPTION
**Pull request by Ottek2**
This pull request translates the Spanish/Italian version of the Boilerplate's UI to an English Version

### CODE REVIEW

**Review by SannitaSSJ — CHANGES_REQUESTED** [review]
Please correct the following on "visualization/index.html":

1. Line 170: please change the title to "Wikicurricula"
2. Line 176: please re-instate "Wikimedistas de Uruguay" as the correct name of the association
3. Line 182: please re-instate "Sofía Doccetti" as the name of the person -- names are untranslatable
4. Line 231: please re-instate "Wikimedia Italia" as the correct name of the association

**Comment by SannitaSSJ** [comment]
@Ottek2 please push the main branch and solve the conflicts, by maintaining as much as possible the structure of the main branch.

**Comment by fernandouval** [comment]
@Ottek2 I have reverted the branch to the main branch. Can you resolve the conflicts and check if there is any improvements in the current code?

**Comment by SannitaSSJ** [comment]
I took the liberty of solving the conflicts, but @fernandouval please review this PR before merging.

File: visualization/index.html [hunk 1]
```diff
@@ -113,16 +113,16 @@ <h1>
...
-								<option name="images" value="7" aria-label="número de imágenes">número de imágenes</option>
-								<option name="reference" value="8" aria-label="referencias">referencias</option>
+								<option name="article" value="1" aria-label="article name">article name</option>
+								<option name="publication" value="2" aria-label="publication date">publication date</option>
+								<option name="size" value="3" aria-label="article size">article size</option>
+								<option name="discussion" value="4" aria-label="talk page size">talk page size</option>
+								<option name="incipit" value="5" aria-label="introduction size">introduction size</option>
+								<option name="issue" value="6" aria-label="number of alerts">number of alerts</option>
+								<option name="images" value="7" aria-label="number of images">number of images</option>
+								<option name="reference" value="8" aria-label="references">references</option>
```
Comment (wikicurricula-uy) [inline visualization/index.html:125]: this should be "number of references" to maintain the same style as in the other options

File: visualization/index.html [hunk 2]
```diff
@@ -135,39 +135,39 @@ <h1>
 			<section id="legend" class="container">
 				<div id="dv_infobox" class="row">
 					<div class="column_3">
-						<h2>Cómo leer la visualización</h2>
+						<h2>How to read the display</h2>
```
Comment (wikicurricula-uy) [inline visualization/index.html:138]: I'm not sure display is the best translation... visualization? chart?

File: visualization/index.html [hunk 3]
```diff
@@ -135,39 +135,39 @@ <h1>
...
 						<p>
-							La visualización muestra los artículos que identificamos como relacionados con la currícula del tercer ciclo de educación básica integrada (ex ciclo básico) de Uruguay.
+							The visualization shows the articles that we identify as related to the curriculum of the third cycle of integrated basic education (former basic cycle) of Uruguay.
```
Comment (wikicurricula-uy) [inline visualization/index.html:140]: we may remove "(former basic cycle)" since it does not add context for English-speaking users

**Review by wikicurricula-uy — APPROVED** [review]

File: visualization/index.html (hunk 2 above)
Comment (SannitaSSJ) [reply visualization/index.html]: Fixed it with "visualization", since it's used in the very next line.

File: visualization/index.html (hunk 3 above)
Comment (SannitaSSJ) [reply visualization/index.html]: Done.

**Review by fernandouval — APPROVED** [review]
Translations seem to be adjusted for the current text, thanks.

**Pull request merged**